test/core/shrink_algo_logs/int_smaller_209609_qcheck2.expected

```diff
@@ -0,0 +1,4 @@
+holds 0
+holds 1
+holds 2
+fails 4611686018427387903
```

Comment:
it's a bit of a problem that this doesn't not shrink, or am I missing something ?

Reply:
I characterized it as "debatable" myself when I added the tests originally 😉
  https://github.com/c-cube/qcheck/pull/153#issuecomment-898731820
incl. what I saw as arguments for and against.

There's also a level-headed reply from @sir4ur0n here:
  https://github.com/c-cube/qcheck/pull/153#issuecomment-900122352